types/vote_set.go

```diff
@@ -541,8 +541,10 @@ func (voteSet *VoteSet) MakeCommit() *Commit {
 	}
 
 	// For every validator, get the precommit
-	votesCopy := make([]*Vote, len(voteSet.votes))
-	copy(votesCopy, voteSet.votes)
+	votesCopy := make([]*CommitSig, len(voteSet.votes))
```

Comment:
No longer a copy in a strict sense. Rename to `precommits`?

Reply:
Is this copy really any different than the original?

Reply:
Not sure I understand what you're getting at. You're just saying we should change the name since its commitsig now instead of votes? Or are you suggesting that using the for loop instead of the `copy` is leading to some different underlying behaviour?

Reply:
> You're just saying we should change the name since its commitsig now instead of votes?

yes